### DISCUSSION TITLE
How to import Hyper PLA

### DISCUSSION THREAD

**Discussion by IQ-Thinker** [OrcaSlicer/OrcaSlicer#2984] [Q&A]
I have a Creality K1. I want to use the Hyper PLA for faster printing, but there is no JSON file available. I want to use OrcaSlicer over the default software because of the tree supports (Creality Slicer doesn't have them for some reason) and ironing feature. A model I have would take over 4 hours to print with OrcaSlicer default PLA but will take less than 3 hours with Hyper PLA. I tried making my own JSON, but I simply can't. I only have a .cxfilament export from Creality Print. It is attached.

`material_diameter=1.75
speed_slowtofast_slowdown=false
speed_slowtofast_slowdown_revise_distance=55
speed_slowtofast_slowdown_revise_speed_factor=30
speed_slowtofast_slowdown_revise_acceleration=1500
default_material_print_temperature=220
build_volume_temperature=28
material_print_temperature=220
material_print_temperature_layer_0=220
material_initial_print_temperature=220
material_final_print_temperature=220
default_material_bed_temperature=45
material_bed_temperature=45
material_bed_temperature_layer_0=45
material_flow_dependent_temperature=false
material_flow_temp_graph=[[5.0,210],[21.0,220],[23.0,230]]
material_surface_energy=100
material_shrinkage_percentage=100.0
material_shrinkage_percentage_z=100.0
material_break_preparation_temperature=220
material_flow_ratio=95
material_flow=100
wall_material_flow=100
wall_0_material_flow=100
wall_x_material_flow=100
skin_material_flow=100
infill_material_flow=100
skirt_brim_material_flow=100
support_material_flow=100
support_interface_material_flow=100
support_roof_material_flow=100
support_bottom_material_flow=100
prime_tower_flow=100
material_flow_layer_0=100
material_standby_temperature=180
material_name=Hyper PLA_1.75
material_brand=Generic
material_color=""
material_density=1.24
material_max_volumetric_speed=23
filament_cost=20
filament_weight=1000
filament_length=0
cost_per_meter=0.00
material_description=""
material_pressure_advance=false
material_advance_length=0.04
retraction_enable=true
retraction_wipe=true
wipe_length=2
before_wipe_retraction_amount_percent=70
retract_at_layer_change=false
retraction_amount=0.5
retraction_speed=40
retraction_retract_speed=40
retraction_prime_speed=40
retraction_extra_prime_amount=0
travel_avoid_distance=0.625
retraction_hop_enabled=true
retraction_hop=0.2
cool_fan_enabled=true
cool_fan_speed=100
cool_fan_speed_min=50
cool_fan_speed_max=100
cool_min_layer_time_fan_speed_max=15
cool_overhang_bridge_force_cooling=true
cool_overhang_threshold=50
cool_overhang_fan_speed=100
cool_small_feature_fan_speed_factor=50
bridge_fan_speed=20
bridge_fan_speed_2=0
bridge_fan_speed_3=0
support_supported_skin_fan_speed=100
cool_fan_speed_0=0
cool_fan_full_at_height=0.50
cool_fan_full_layer=3
cool_min_layer_time=5
cool_infill_speed_slowdown_first=false
cool_min_speed=20
cool_lift_head=false
cool_cds_fan_enable=true
cool_cds_fan_speed=50
cool_special_cds_fan_speed=80
cool_cds_fan_start_at_height=0.5
cool_chamber_fan_enable=true
material_type=PLA
`
Please tell me any ideas that you have!

**Comment by ZhaoZhen711** [comment]
tree supports and ironing are available in Creality Print
double click your Parameter config, then enable the Advanced toggle
![image](https://github.com/SoftFever/OrcaSlicer/assets/45798807/da93e7f3-95e7-4e67-9330-6c802accd54b)
